CHANGELOG.md

```diff
@@ -1,3 +1,431 @@
+ ([#651](https://github.com/kinvolk/lokomotive/pull/651)).
+
+- Update contour configuration reference for usage with AWS
+   ([#674](https://github.com/kinvolk/lokomotive/pull/674)).
+
+- Add documentation related to the usage of `clc_snippets` for Packet and AWS
+    ([#657](https://github.com/kinvolk/lokomotive/pull/657)).
+
+- Improve documentation on using remote
+    [backends](https://github.com/kinvolk/lokomotive/blob/v0.3.0/docs/configuration-reference/backend/s3.md)
+    ([#670](https://github.com/kinvolk/lokomotive/pull/670)).
+
+- How to guide for setting up monitoring on Lokomotive
+    ([#480](https://github.com/kinvolk/lokomotive/pull/480)).
+
+- Add `codespell` section in [development
+    documentation](https://github.com/kinvolk/lokomotive/blob/v0.3.0/docs/development/README.md)
+    ([#700](https://github.com/kinvolk/lokomotive/pull/700)).
+
+#### Bugfixes
+
+- Remove contour ingress workaround(due to an [upstream issue](https://github.com/projectcontour/contour/issues/403))
+  for ExternalDNS ([#635](https://github.com/kinvolk/lokomotive/pull/635)).
+
+#### Development
+
+- Must use consistent build tags for all platforms for CI
+    ([#600](https://github.com/kinvolk/lokomotive/pull/600))
+    & ([#676](https://github.com/kinvolk/lokomotive/pull/676)).
+
+- Disable marking releases as `pre-releases` ([#678](https://github.com/kinvolk/lokomotive/pull/678)).
+
+- Show release values in `--verbose` mode only
+    ([#627](https://github.com/kinvolk/lokomotive/pull/627)).
+
+- Change NewClientset to create Kubernetes Clientset from kubeconfig file
+content, instead of from file path
+([#628](https://github.com/kinvolk/lokomotive/pull/628)).
+
+- Remove provider aliases from go templates
+    ([#617](https://github.com/kinvolk/lokomotive/pull/617)).
+
+
+#### Miscellaneous
+
+- Following flatcar-linux/Flatcar#123, Flatcar 2513.1.0 for ARM contains the dig
+    binary so the workaround is no longer needed
+    ([#703](https://github.com/kinvolk/lokomotive/pull/703)).
+
+- Make lint target fail upon invalid git configuration locally. ([#488](https://github.com/kinvolk/lokomotive/pull/488)).
+
+- Improve error message for `wait-for-dns` output
+    ([#735](https://github.com/kinvolk/lokomotive/pull/735)).
+
+- Add `codespell` target to Makefile and `all` target ([#661](https://github.com/kinvolk/lokomotive/pull/661)).
+
+### Upgrading from v0.2.1
+
+#### Configuration syntax changes
+
+There have been some minor changes to the configurations of following components:
+* contour
+* rometheus-operator.
+
+Please make sure new the configuration structure is in place before the upgrade.
+
+##### Contour component
+
+Optional `ingress_hosts` variable is now removed.
+
+old:
+
+```hcl
+component "contour" {
+  .
+  .
+  ingress_hosts = ["*.example.lokomotive-k8s.net"]
+}
+```
+
+new:
+
+```hcl
+component "contour" {
+  .
+  .
+}
+
+```
+
+##### Prometheus-operator component
+
+* Variable `external_url` is now removed.
```

Comment:
Hm, this is actually moved to `ingress.host` field. I think we should mention that to the user. And the valid value changed: now `http://` needs to be removed.

Reply:
```
Variable `external_url` is now removed and now configured under
    `prometheus.ingress.host`. Remove the http scheme when configuring.
```